specifyweb/frontend/js_src/lib/components/Preferences/CollectionDefinitions.tsx

```diff
@@ -29,7 +29,7 @@ export const collectionPreferenceDefinitions = {
             title: 'Defines if preparation stats include total',
```

Comment:
is this prop just for preparation stats? in that case, perhaps a more descriptive name than `showTotal` would do?
remember that we can't change pref names easily once released so have to be smart about them now.

i.e `showPreparationsTotal` or something like that that won't cause confusion and collisions in the future

Reply:
Resolved